W-14379658 Update imp-implementing-apis.adoc (#239)
mulesoft/docs-code-builder: modules/ROOT/pages/imp-implementing-apis.adoc

//LOGO (web, desktop, or both)
// include::partial$acb-ide-logos.adoc[tags="both-ides"]

> Use Anypoint Code Builder to start an API implementation project by importing the API spec and scaffolding it into an interface to implement and develop within a Mule app.

Author: ```suggestion
Use Anypoint Code Builder to start an API implementation project by importing the API spec and scaffolding it into an interface to implement and integrate within a Mule app.
```

> Use Anypoint Code Builder to start an API implementation project by importing the API spec and scaffolding it into an interface for your implementation and integration projects (Mule apps).

Author: ```suggestion
Use Anypoint Code Builder to start an API implementation project by importing the API spec and scaffolding it into an interface to implement and develop within a Mule app.
```
